GEON-PPANG-iOS/Presentation/Common/StackView/IconLabelStackView.swift

```diff
@@ -102,9 +102,17 @@ final class IconLabelStackView: UIStackView {
     private func setLayout() {
         
         self.addArrangedSubview(icon)
-        icon.snp.makeConstraints {
-            $0.top.leading.equalToSuperview().inset(0.5)
-            $0.size.equalTo(icSize)
+        if iconType == .notice {
+            icon.snp.makeConstraints {
+                $0.top.leading.equalToSuperview()
+                $0.size.equalTo(icSize)
+            }
+        } else {
+            icon.snp.makeConstraints {
+                $0.top.equalToSuperview().inset(0.5)
+                $0.leading.equalToSuperview()
+                $0.size.equalTo(icSize)
+            }
         }
```

Comment:
p3:
```suggestion
        icon.snp.makeConstraints {
            $0.top.equalToSuperview().inset(self.iconType == .notice ? 0 : 0.5)
            $0.leading.equalToSuperview()
            $0.size.equalTo(icSize)
        }
```
요로코롬 코드 양 줄일 수 있을듯 !!

Reply:
[247dca9](https://github.com/GEON-PPANG/GEON-PPANG-iOS/pull/203/commits/247dca90b5fe926aca8226da9af53871919e2006)